src/browser/process.rs

```diff
@@ -170,6 +179,14 @@ impl Process {
             args.extend(&["--headless"]);
         }
 
+        let extension_args: Vec<String> = launch_options
+            .extensions
+            .iter()
+            .map(|e| format!("--load-extension={}", e.to_str().unwrap()))
```

Comment:
This could be something to the tune of `let a = String::new("--load-extension").into(); a.push(e)` to avoid `e` from going through string.